web_programming/reddit.py

```diff
@@ -34,7 +32,7 @@ def get_data(
     """
     response = requests.get(f"https://reddit.com/r/{sub}/{age}.json?limit={limit}")
     if response.raise_for_status():
-        raiseExceptions
+        raise requests.HTTPError
```

Comment:
In this case, it is important to raise exception otherwise if the `response` is a 429 (too many requests) , then `response.json()` would make an empty json(and not call an error). This would be problematic because then it would give a `Keyerror` and a false impression that error is caused in Line42 when in reality, the response returned an HTTPError.